cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -463,21 +421,11 @@ func generateMountsModeRun(mst *initramfsMountsState) error {
 
 	// 4.2 check if base is mounted
 	// 4.3 check if kernel is mounted
```

Comment:
these step comments needs to be updated as we mount, not check if mounted anymore